File: src/main/java/com/talenco/lotterysystem/service/WinningResultService.java
package com.talenco.lotterysystem.service;


import com.talenco.lotterysystem.POJO.WinningResult;
import com.talenco.lotterysystem.entity.WinningResultSetting;

import java.util.List;

public interface WinningResultService {

    List<WinningResult> getAll();

    List<WinningResultSetting> getAllByGid();

    List<WinningResult> getOne(int gid);

    void delete(int[] ids);

    void deleteAll();
}

File: src/main/webapp/js/service/winningResult_service.js
app.service('winningResultService', function ($http) {

    this.getAll = function () {
        return $http.get('../WinningResultController/getAll');
    };


    this.insert = function (entity) {
        return $http.post('../WinningResultController/insert', entity)
    };

    this.update = function (entity) {
        return $http.post('../WinningResultController/update', entity)
    };

    this.delete = function (ids) {
        return $http.post('../WinningResultController/delete?ids=' + ids)
    };

    this.deleteAll = function () {
        return $http.post('../WinningResultController/deleteAll')
    };


});
